### PR TITLE
[TASK] Create Fund page for SGD (RT-3044)

### PR DESCRIPTION
**Pull request by Mesrop**

### CODE REVIEW

**Comment by mrajvanshy** [comment]
LGTM

**Comment by MatthewPhinney** [comment]
LGTM

File: src/js/tabs/sgd.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+     $scope.save_account = function() {
+        $scope.loading = true;
+
+        var amount = ripple.Amount.from_human(
+            Options.gateway_max_limit + ' ' + 'SGD', {
+                reference_date: new Date(+new Date() + 5 * 60000)
+            }
+        );
+
+        amount.set_issuer("r9Dr5xwkeLegBeXq6ujinjSBLQzQ1zQGjH");
```
Comment (jublo) [inline src/js/tabs/sgd.js]: We prefer single quotes `'`.

File: src/js/tabs/sgd.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                  $scope.tx_result = 'malformed';
+                  break;
+                case 'ter':
+                  $scope.tx_result = 'failed';
+                  break;
+                case 'tec':
+                  $scope.tx_result = 'failed';
+                  break;
+                case 'tel':
+                  $scope.tx_result = "local";
```
Comment (jublo) [inline src/js/tabs/sgd.js]: See above.

File: src/js/tabs/sgd.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+                  break;
+                case 'tep':
+                  console.warn('Unhandled engine status encountered!');
+              }
+              if ($scope.tx_result == "cleared") {
```
Comment (jublo) [inline src/js/tabs/sgd.js]: Ditto.

File: src/js/tabs/sgd.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+            console.log(err);
+            return;
+          }
+          $scope.mode = 'granting';
+          tx.secret(secret);
+          tx.submit();
+        });
+      };
+      $scope.$watch('lines', function() {
+        if ($scope.lines['r9Dr5xwkeLegBeXq6ujinjSBLQzQ1zQGjHSGD']) {
```
Comment (jublo) [inline src/js/tabs/sgd.js]: Can we written in dot notation.

**Comment by mrajvanshy** [comment]
I am connected to the Ripple Singapore gateway but it shows as 'Not connected' on the SGD fund page 
image

File: src/jade/tabs/sgd.jade [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+          button.btn.btn-large(ng-show="showInstructions && sgdConnected", type="submit", ng-click="toggle_instructions()", l10n) Hide instructions
+          button.btn.btn-large(ng-show="sgdConnected && !showInstructions", type="submit", ng-click="toggle_instructions()", l10n) Show instructions
+
+        .connect.col-xs-5.col-sm-7.col-md-8(ng-hide="loading")
+          .description(ng-show="!sgdConnected && account.Balance", l10n) 
+            i.fa.fa-times
+            |  Not connected
+          .description(ng-show="sgdConnected && account.Balance", l10n)
+            i.fa.fa-check
+            |  Not connected
```
Comment (mrajvanshy) [inline src/jade/tabs/sgd.jade]: Should say 'Connected' not 'Not connected'

**Comment by annatonger** [comment]
I'm seeing the same issue as Malika. 

Also some word edits for the instructions. RippleSingapore needs a space in between, so it's Ripple Singapore.

TO INITIATE SINGAPORE DOLLAR DEPOSITS AND WITHDRAWALS
Register or login to your **RippleSingapore** account. You will need:
A valid non-US, government-issued ID (passport, driver’s license, etc)
Proof of address
Verification is required for deposits or withdrawals
DEPOSIT OPTIONS
Users can deposit SGD by entering bank credentials and initiating a bank wire transfer from user’s bank account to Ripple Singapore **add a period at the end of this sentence**
WITHDRAWAL OPTIONS
Users can withdraw SGD by sending funds from their Ripple address to **RippleSingapore’s** Ripple address. **RippleSingapore** will send the funds to a user’s bank account using a bank wire transfer**add a period at the end of this sentence**

**Comment by Mesrop** [comment]
@annatonger @mrajvanshy can you check please

**Comment by annatonger** [comment]
You still need to add periods at the end of the sentences

**Comment by Mesrop** [comment]
@annatonger sorry. done.

**Comment by mrajvanshy** [comment]
LGTM

**Comment by annatonger** [comment]
LGTM

**Pull request merged**